cmd/kubelet/app/server_windows.go

```diff
@@ -0,0 +1,77 @@
+// +build windows
+
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import (
+	"fmt"
+	"os/user"
+
+	"golang.org/x/sys/windows"
+)
+
+func isAdmin() bool {
+	u, err := user.Current()
+	if nil != err {
+		return false
+	}
+
+	ids, err := u.GroupIds()
+	if nil != err {
+		return false
+	}
+	for i := range ids {
+		// BUILTIN\ADMINISTRATORS
+		if "S-1-5-32-544" == ids[i] {
+			return true
+		}
+	}
+
+	return false
+}
+
+func checkPermissions() error {
+	//https://github.com/golang/go/issues/28804#issuecomment-505326268
```

Comment:
run `./hack/update-gofmt.sh` to fix some golint problems.

Reply:
Thanks @pacoxu, I've ran that but it doesn't seem to have changed anything, anything specific that I need to correct?